bn-models/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@boatnet/bn-models",
-  "version": "0.1.9",
+  "version": "0.1.10",
```

Comment:
you'll probably want to make this 0.2.1, i pushed 0.2.0 yesterday https://www.npmjs.com/package/@boatnet/bn-models

Reply:
That's weird, I re-pulled from github master and it didn't give me that commit. I'll have to rebase again.

Reply:
oh yeah 0.1.9 is the latest version in the code, I didn't push my version update change, so you can just change the code manually don't think you'll need to do a rebase

Reply:
I updated https://github.com/nwfsc-fram/boatnet-module/blob/master/CONTRIBUTING.md to add a reminder to push and merge your changes to master before publishing to npm